### ISSUE TITLE
Results-per-page dropdown on the search results page should have a better accessible name

### ISSUE THREAD

**Issue by sandbergja** [pulibrary/orangelight#3375]
@rladdusaw, @maxkadel and I found this while testing a different issue in VoiceOver.

### Expected behavior
When you open the "20 results per page" dropdown, VoiceOver on Firefox does not give a confusing name to the dropdown.

### Actual behavior

When you open the dropdown in firefox, voiceover announces "menu 20 per page 4 items".

### Steps to replicate

1. Open the catalog in firefox
2. Perform a search
3. Turn on voiceover
4. Tab until you get to the "20 per page" button
5. Press enter to open the dropdown
6. Press control + option + Right arrow to access the dropdown
7. Notice that voiceover announces "menu 20 per page 4 items", rather than something more expected like "menu Results per page 4 items"

### Impact of this bug

Adds confusion to using this with a screen reader.

### Implementation notes, if any

This issue is also present in Blacklight itself, so we may want to open a PR there.

**Comment by kevinreiss** [comment]
Likely an upstream PR to Blacklight

**Comment by kevinreiss** [comment]
Need to come to a clear recommendation on what language is clear.
